config/sepc/retina_r50v1b_fpn_1x.py

```diff
@@ -0,0 +1,238 @@
+from models.retinanet.builder import RetinaNet as Detector
```

Comment:
What's the point of replicating an existing config in sepc

Reply:
Hi, what this mean?

Reply:
This is already a existing retina_r50v1b_fpn_1x.py in the resnet_v1b config folder, why duplicate it here?

Reply:
Hi, first I want to change fp16 from False (in resnet_v1b config folder) to True to make 4 imgs/per_gpu training possible in SEPC setting. Offically SEPC config in mmdet suggests >=4imgs/per_gpu when using ibn. Then I use sync_bn (not fixed_bn) for both backbone and ibn considering the case when ibn is used. In the official mmet fixed_bn is used for only backbone not ibn, I guess. Of course here I have no experiments on fixed_bn when using ibn.